ColourDetection[] detections
uint8 canMoveForwards
uint8 canTurnRight
uint8 canTurnLeft
